### ISSUE TITLE
Support for Melodic

### ISSUE THREAD

**Issue by Hinsow** [furushchev/respeaker_ros#20]
Hello, I would like to know if you will upgrade your drive to support Melodic?  

Thank you for you support

**Comment by nordic1814** [comment]
Yes Also I am trying, do run on Melodic, but getting errors hope you will.

**Comment by nordic1814** [comment]
It's now publishing the sound_direction topic on ROS Melodic for Ubuntu 18.04, so getting the angels now form the mic. But still got a error RED on the speech_recognition.
